aws/logs_monitoring/tools/integration_test.sh

```diff
@@ -11,22 +11,28 @@ set -e
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 cd $DIR
 
-PYTHON_VERSIONS=("python3.7")
```

Comment:
Would it have been easier to just add python3.8 to this array?

Reply:
I wanted to allow us to run the two different versions in parallel (I set up the GitHub actions accordingly). So, it made more sense to have an argument for this script rather than having the script run both in series.

Reply:
Ahh, that makes sense. LGTM